src/qibo/derivative.py

```diff
@@ -11,6 +184,7 @@ def parameter_shift(
     initial_state=None,
     scale_factor=1,
     nshots=None,
+    cdr_params=None,
```

Comment:
I would drop this. I think the mitigation conf can be involved into the optimization module, or in the VQC module if we want to create one.

Reply:
Unfortunately the error mitigation can't simply be applied to the end result. If the function $g$ handles our error mitigation, we have: $C * (g(shift_forward) - g(shift_backward)) \noteq g(C * (g(shift_forward) - g(shift_backward))$